packages/main/src/plugin/util/exec.ts

```diff
@@ -0,0 +1,145 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ChildProcessWithoutNullStreams } from 'child_process';
+import { spawn } from 'child_process';
+import type { RunError, RunOptions, RunResult } from '@podman-desktop/api';
+import { isMac, isWindows } from '../../util.js';
+
+export const macosExtraPath = '/usr/local/bin:/opt/homebrew/bin:/opt/local/bin:/opt/podman/bin';
+
+export function exec(command: string, args?: string[], options?: RunOptions): Promise<RunResult> {
+  let env = Object.assign({}, process.env);
+
+  if (isMac() || isWindows()) {
+    const installationPath = getInstallationPath();
+    env.PATH = installationPath + (env.PATH ? (isWindows() ? ';' : ':') + env.PATH : '');
```

Comment:
It looks to me that `getInstallationPath` already add the `env.PATH`, no? 
Should it just be `env.PATH = installationPath`?

Reply:
yep, thanks for pointing this, simplified env initialization